python/ray/serve/tests/test_standalone3.py

```diff
@@ -440,23 +440,35 @@ def test_healthz_and_routes_on_head_and_worker_nodes(
 
     # Setup a cluster with 2 nodes
     cluster = Cluster()
-    cluster.add_node(num_cpus=3)
-    cluster.add_node(num_cpus=3)
+    cluster.add_node(num_cpus=1)
+    cluster.add_node(num_cpus=2)
```

Comment:
How about just setting `num_cpus=0` on the head node so no replicas get scheduled there and all are on worker node? That would simultaneously test that head node keeps returning 200s.

Reply:
I was thinking to test both cases when head node and worker node has replicas and then neither have any. But yea, just make sure all replicas goes to worker node should ensure the stability of the test! Head node will always get 200s regardless